validator/src/types.rs

```diff
@@ -49,6 +59,39 @@ impl std::error::Error for Errors {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct Error {
```

Comment:
I kept with your approach, the Errors struct isn't called ValidationErrors, so why add the prefix for the singular struct?